Sources/PointFree/Auth.swift

```diff
@@ -144,14 +144,14 @@ private func authTokenMiddleware(
   )
   -> IO<Conn<ResponseEnded, Data>> {
 
-    return AppEnvironment.current.fetchAuthToken(conn.data.code)
+    return AppEnvironment.current.gitHub.fetchAuthToken(conn.data.code)
       .flatMap { token in
-        AppEnvironment.current.fetchGitHubUser(token)
-          .map { user in GitHubUserEnvelope(accessToken: token, gitHubUser: user) }
+        AppEnvironment.current.gitHub.fetchUser(token)
```

Comment:
We now scope our environment calls through these bundles, which means we can have both `gitHub.fetchUser` and `database.fetchUser`.